lib/net/nforceif/include/lwipopts.h

```diff
@@ -437,7 +437,13 @@
 #define TCP_QLEN_DEBUG   LWIP_DBG_ON
 #define TCP_RST_DEBUG    LWIP_DBG_ON
 
+#ifdef __cplusplus
+extern "C" {
+#endif
 extern unsigned char debug_flags;
+#ifdef __cplusplus
+}
+#endif
```

Comment:
Wouldn't this file only be included by C code in lwip? When would this ever be compiled by a C++ compiler?

Reply:
It gets pulled in indirectly by lwip headers.

Reply:
I see. lwip seems to have this C++ handling in each header *after* `#include` lines.
Resolved.